source/deployment/sso-saml-ldapsync.rst

```diff
@@ -29,3 +29,7 @@ Once the synchronization with AD/LDAP is enabled, user attributes are synchroniz
     
     If a user is deactivated from SAML, their session won't expire until they're deactivated from AD/LDAP. However, they won't be able to log back in to Mattermost.
  
+   .. note::
```

Comment:
@crspeller Can you help review accuracy of the new changes in our SAML-LDAP sync docs

Reply:
Yup sounds good.